fixtures/fail-versions.github

```diff
@@ -1,3 +1,4 @@
 # FAILURE
-# *ERROR* servant-client-core is missing tested-with annotations for: ghc-7.8.1,ghc-7.8.2,ghc-7.8.3,ghc-7.8.4
-# *ERROR* servant-foreign is missing tested-with annotations for: ghc-7.8.1,ghc-7.8.2,ghc-7.8.3,ghc-7.8.4
+# *ERROR* servant-client-core is missing tested-with annotations for: ghc-7.8.1, ghc-7.8.2, ghc-7.8.3, ghc-7.8.4
+# *ERROR* servant-foreign is missing tested-with annotations for: ghc-7.8.1, ghc-7.8.2, ghc-7.8.3, ghc-7.8.4
+(Consider option --jobs-selection any)
```

Comment:
Don't write (edtit) files by hand.

Use `cabal run golden -- --accept` (the `--accept` will accept and write differences).

Reply:
A thanks, didn't know that trick.